src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,66 @@
+package duke;
+
+public class Ui {
+
+    static String divider = "____________________________________________________________";
+
+    public void sendStartMessage() {
+        System.out.println("Hello I'm Karen. What do you want??\n" + divider);
+    }
+    public void sendMarkedMessage(Task task) {
+        System.out.println(divider);
+        System.out.println("Oh you did a task. Congratulations.\n" + task + "\n" + divider);
+    }
+    
+    public void sendUnmarkedMessage(Task task) {
+        System.out.println(divider);
+        System.out.println("Hmm make up your mind maybe??.\n" + task + "\n" + divider);
+    }
+    public void sendFailureMessage() {
+        System.out.println(divider);
+        System.out.println("What's that??");
+        System.out.println(divider);
+    }
+    
+    public void sendEmptyTaskMessage() {
+        System.out.println(divider);
+        System.out.println("Empty task? Are you kidding me??");
+        System.out.println(divider);
```

Comment:
```suggestion
        String s = divider + "\n" 
                + "Empty task? Are you kidding me??" + "\n" 
                + divider;
        System.out.println(s);
```
Would it be better to combine all the messages into one string before printing it out? It may reduce the need to call System.out.println() many times.